dvc/cache.py

```diff
@@ -12,24 +12,25 @@ def __init__(self, project):
         self.project = project
 
         config = project.config._config[Config.SECTION_CACHE]
+        local = config.get(Config.SECTION_CACHE_LOCAL)
 
-        local = config.get(Config.SECTION_CACHE_LOCAL, None)
         if local:
             name = Config.SECTION_REMOTE_FMT.format(local)
             sect = project.config._config[name]
         else:
-            sect = {}
             cache_dir = config.get(Config.SECTION_CACHE_DIR, self.CACHE_DIR)
+            cache_type = config.get(Config.SECTION_CACHE_TYPE)
+            protected = config.get(Config.SECTION_CACHE_PROTECTED)
+
             if not os.path.isabs(cache_dir):
-                cache_dir = os.path.join(project.dvc_dir, cache_dir)
+                cache_dir = os.path.join(os.getcwd(), cache_dir)
```

Comment:
So when you run dvc from different directories, cache_dir is going to be resolved differently. This doesn't look right. I think we should leave it as it is, but make it clear in the docs.

Reply:
That's so true, @efiop; we could also add an extra `../` for relative paths so it starts on top of the `dvc_dir`, but I agree that we should better document this instead of change the current logic.

Could we close this PR?

Reply:
Thanks, @efiop :stuck_out_tongue: !

Here is the PR https://github.com/iterative/dvc/pull/1321

Reply:
@mroutis You could've just re-purposed this one though :slightly_smiling_face: